controller/image.yaml

```diff
@@ -101,6 +101,8 @@ modules:
       version: "1.0-kie"
     - name: os-eap7-launch
       version: "1.0-kie"
+    - name: jboss.container.wildfly.launch.access-log-valve
```

Comment:
this addition is from another commit, I don't think it should be merged together with this one.

Reply:
Fixed